### ISSUE TITLE
Configurator should support c_flags and cxx_flags

### ISSUE THREAD

**Issue by rgrinberg** [ocaml/dune#1771]
https://github.com/ocaml/dune/pull/1700#issuecomment-455242271

**Comment by XVilka** [comment]
Shouldn't be this one changed to a more recent milestone?

**Comment by rgrinberg** [comment]
Indeed. I've removed this for now. It's not that urgent and we should wait for some user reports.
